commands/sunsubscribe.md

```diff
@@ -0,0 +1,7 @@
+Unsubscribes the client from the given shard channels, or from all of them if none is
```

Comment:
```suggestion
Unsubscribes the client from the given shard channels, or from all of them if none is given.
```

Reply:
Thanks @enjoy-binbin. My bad, didn't check the README :|

Reply:
I didn't even see this until recently either. I never check the README :)